bc_obps/service/form_builder_service.py

```diff
@@ -40,18 +43,40 @@ def build_source_type_schema(
     gas_type_enum = []
     # Maps of the gas_type & methodology objects will be passed in the return object so we have the IDs on the frontend.
     gas_type_one_of: Any = {"gasType": {"oneOf": []}}
+
+    gas_type_ids = list(map(lambda config_elt: config_elt.gas_type.id, gas_types))
```

Comment:
Should be the same as: `gas_types.objects.values_list('gas_type_id', flat=True)`